src/streambuffer.h

```diff
@@ -151,6 +151,16 @@ class StreamBuffer
         }
         exit(-1);
     }
+    /*NEW*/
+    inline bool checkForChar(char c){
```

Comment:
We use the official syntax for those constraints, as defined by DIMACS cnf+
See: page 16 in https://pysathq.github.io/docs/pysat.pdf

Reply:
OK, fair enough. Please create a separate function for parsing these, let's not re-use clause parsing for this. Just copy-paste and adapt to your needs. Currently, it's very inelegant with the boolean switch. Thanks!

Reply:
The problem is that, when we start parsing a line, we do not know yet if it is going to be a clause or a cardinality constraint. With a prefix at the beginning of the line, it would be easy to decide, but with the official syntax, I do not know how to get rid of the boolean switch.

Reply:
I don't think it's in any way an "official" syntax. Someone created it, now it's "official". No body of researchers agreed it's good, as far as I know. Anyway, I understand, Let's leave it as-is. 

Please, for the sake of all that is holy on this planet, remove those "/NEW/" things, they are ugly :) Thanks!

Reply:
All the NEWs have already been removed before.